spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -310,30 +313,33 @@ def step(  # noqa
                 self.say("Grasping " + self.target_obj_name)
 
                 # The following cmd is blocking
-                success = self.attempt_grasp()
+                success = self.attempt_grasp(
+                    action_dict.get("enable_pose_estimation", False)
+                )
                 if success:
                     # Just leave the object on the receptacle if desired
                     if self.config.DONT_PICK_UP:
                         self.say("open_gripper in don't pick up")
                         self.spot.open_gripper()
                     self.grasp_attempted = True
-                    arm_positions = np.deg2rad(self.config.PLACE_ARM_JOINT_ANGLES)
+                    # arm_positions = np.deg2rad(self.config.PLACE_ARM_JOINT_ANGLES)
                 else:
                     self.say("BD grasp API failed.")
                     self.spot.open_gripper()
                     self.locked_on_object_count = 0
-                    arm_positions = np.deg2rad(self.config.GAZE_ARM_JOINT_ANGLES)
+                    # arm_positions = np.deg2rad(self.config.GAZE_ARM_JOINT_ANGLES)
                     time.sleep(2)
 
+                # TODO: This changes the behavior of post grasping
```

Comment:
I added todo here to discuss the pose grasping behavior as this substantially changes the behavior